braket_features/Getting devices and checking device properties.ipynb

```diff
@@ -0,0 +1,340 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting a device and checking device properties\n",
+    "\n",
+    "In this tutorial, we introduce methods to get a device to run your circuits by either using `get_devices()` or specifying the device directly. We also show what properties of a device you can find for each device."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# general imports\n",
+    "from braket.aws import AwsDevice\n",
+    "from braket.devices import LocalSimulator"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using get_devices\n",
+    "We can get a device (including the cloud-based simulator and the QPUs) by using the `get_devices()` functions. The corresponding devices can be found with one or more of the following filtering criteria: device arn, name, type, status and provider_name. We can also sort devices in order. We give examples for each of the cases in the following cells."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[Device('name': Advantage_system1.1, 'arn': arn:aws:braket:::device/qpu/d-wave/Advantage_system1),\n",
+       " Device('name': Aspen-8, 'arn': arn:aws:braket:::device/qpu/rigetti/Aspen-8),\n",
+       " Device('name': DW_2000Q_5, 'arn': arn:aws:braket:::device/qpu/d-wave/DW_2000Q_5),\n",
```

Comment:
What's this device?

Reply:
the old Dwave device. It is always offline, but still available in the list.